test_util/cluster_api.py

```diff
@@ -316,42 +316,24 @@ def get_node_url(self, node, port=None):
             netloc = '{}:{}'.format(node, port)
         return '{}://{}'.format(self.scheme, netloc)
 
-    def cluster_api_request(self, method, path, host_url=None, node=None, port=None, **kwargs):
-        """
-        Performs the same specialized request as ClusterApi but allows setting node
-        Args:
-            see ClusterApi for descriptions
-            node: host string corresponding to the ones stored in this object
-            port: if node is given, then a port can also be specified
-        """
-        if node is not None:
-            assert host_url is None, 'Cannot set both node and host_url'
-            host_url = self.get_node_url(node, port=port)
-        return self.api_request(method, path, host_url=host_url, **kwargs)
-
-    get = functools.partialmethod(cluster_api_request, 'get')
-    post = functools.partialmethod(cluster_api_request, 'post')
-    put = functools.partialmethod(cluster_api_request, 'put')
-    delete = functools.partialmethod(cluster_api_request, 'delete')
-    options = functools.partialmethod(cluster_api_request, 'options')
-    head = functools.partialmethod(cluster_api_request, 'head')
-    patch = functools.partialmethod(cluster_api_request, 'patch')
-    delete = functools.partialmethod(cluster_api_request, 'delete')
-
     def get_client(self, path, default_headers=None):
```

Comment:
nit: Could we just have this be a method of ApiClient? From what I can tell there's nothing cluster specific about it.